sdk/compute/mgmt-v2020_10_01_preview/src/main/java/com/microsoft/azure/management/compute/v2020_10_01_preview/VirtualMachineRunCommands.java

```diff
@@ -31,7 +33,7 @@
      * @throws IllegalArgumentException thrown if parameters fail the validation
      * @return the observable for the request
      */
-    Observable<LocationVirtualMachineRunCommand> getAsync(String location, String commandId);
+    Observable<RunCommandDocumentInner> getAsync(String location, String commandId);
```

Comment:
Quick fix for exposing the inner class instead of adding interface manually in lite SDK.

Reply:
What is the problem of using the non inner object?

Reply:
No interface and impl class generated for that inner.

Reply:
Interface seems generated `LocationVirtualMachineRunCommand`